src/plugins/components/layout/centered-container.js

```diff
@@ -2,36 +2,46 @@
  * Centered container.
  */
 module.exports = function () {
-  return function ({ addComponents, theme }) {
+  return function ({ addUtilities, theme }) {
     // Find and set the padding based on the screen margins
     const screens = theme('screens');
-    const padding = {};
+    const maxWdiths = {};
+    let largestScreen = 0;
+    let largestGutter = 0;
+    let smallestGutter = 999999;
```

Comment:
Just shy of a million?

Reply:
![dr evil](https://media.tenor.com/3Osoz2-u6y8AAAAC/dr-evil-austin-powers.gif)